### ISSUE TITLE
Add option to disable web hooks in dev mode

### ISSUE THREAD

**Issue by levinelson** [craftcms/webhooks#4]
Hi, love the plugin! It'd be awesome if there was an easy switch for disabling the web hooks while in dev mode, so I don't keep sending all sorts of test data to the web hook.

**Comment by brandonkelly** [comment]
The next Craft CMS release will have a new `disabledPlugins` config setting, which you can use to disable the Webhooks plugin entirely in Dev environments:

```php
// -- config/general.php --
'dev' => [
    'disabledPlugins' => ['webhooks'],
],
```

**Comment by devinellis** [comment]
Alternatively, maybe the webhook payload could include the environment name? It would allow for deeper integrations (e.g. separate channels in Slack)

**Comment by brandonkelly** [comment]
@devinellis Can you post as a separate issue?

**Comment by carlcs** [comment]
@brandonkelly I’ve just read the issue in the CMS repo about the problem that can occur when disabling plugins on an environment basis (https://github.com/craftcms/cms/issues/9576). It sounds like we need a better solution for disabling webhook calls.

**Comment by brandonkelly** [comment]
@carlcs Yeah good point. Just released v3.4.1 with a new `disableAllWebhooks` setting you can set to `false` from `config/webhooks.php`:

```php
use craft\helpers\App;

return [
    'disableAllWebhooks' => (bool)App::env('DISABLE_WEBHOOKS'),
    // ...
];
```
